python/doc/theory/numerical_methods/nataf_transformation.rst

```diff
@@ -3,40 +3,35 @@
 Generalized Nataf Transformation
 --------------------------------
 
-The Generalized Nataf transformation is an isoprobabilistic
-transformation (refer to ) which is used under the following context :
-:math:`\vect{X}` is the input random vector, :math:`F_i` the
-cumulative density functions of its components and :math:`C` its
-copula, which is supposed to be elliptical.
+The Generalized Nataf transformation is an :ref:`isoprobabilistic_transformation` which is
+used under the.. _rosenblatt_transformation following context: the input random vector is :math:`\vect{X}` with marginal
```

Comment:
looks like there was a chunk of '.. _rosenblatt_transformation' inserted

Reply:
yes!